modules/nw-sriov-network-attachment.adoc

```diff
@@ -28,20 +28,35 @@ apiVersion: sriovnetwork.openshift.io/v1
 kind: SriovNetwork
 metadata:
   name: <name> <1>
-  namespace: sriov-network-operator <2>
+  namespace: openshift-sriov-network-operator <2>
 spec:
   networkNamespace: <target_namespace> <3>
   ipam: |- <4>
     ...
   vlan: <vlan> <5>
   resourceName: <sriov_resource_name> <6>
+  linkState: <link_state> <7>
+  maxTxRate: <max_tx_rate> <8>
+  minTxRate: <mix_rx_rate> <9>
+  vlanQoS: <vlan_qos> <10>
+  spoofChk: <spoof_check> <11>
+  trust: <trust_vf> <12>
+  capabilities: <capabilities> <13>
+  
 ----
 <1> Replace `<name>` with a name for the CR. The Operator will create a NetworkAttachmentDefinition CR with same name.
 <2> Specify the namespace where the SR-IOV Operator is installed.
-<3> Replace `<target_namespace>` with the namespace where the NetworkAttachmentDefinition CR will be created.
-<4> Specify a configuration object for the ipam CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
-<5> Replace `<vlan>` with a Virtual LAN (VLAN) ID for the additional network. The integer value must be from `0` to `4095`. The default value is `0`.
+<3> Optional. Replace `<target_namespace>` with the namespace where the NetworkAttachmentDefinition CR will be created.
```

Comment:
Add default  `target_namespace` if not specified? like default `target_namespace` is `openshift-sriov-network-operator` if not specified or what else it would be.